### DISCUSSION TITLE
Displaying %mem% or %var% in WebGui

### DISCUSSION THREAD

**Discussion by kuhnino** [arendst/Tasmota#14489] [Q&A] [answered]
Hi there
Using esp82166-01 

Question can I modify the WebGUI to display content of mem<X> or var<X> ?
As this is a calculated or a value from a second device received by websend.

Any idea to bring me on the right track is much apreciated

**Comment by sfromis** [comment, marked as answer]
Not really. The web UI is meant for admin/debug, not intended as a "primary point of operation", all that is expected to happen via a backend dashboard system, and Tasmota does not provide for doing web design on the web UI. To a very limited extent, you can "cheat" putting something inside a WebButton, but that is not updated without a browser tab refresh. With ESP32, Berry does allow you to program things to add to the web UI, and the scripting language also something in that direction, IIRC.

**Comment by kuhnino** [comment]
Fair enough

I though something like using a virtual sensor and then "modifying" the shown entries could be possible

**Reply by sfromis** [reply]
That's an option with Berry on ESP32
